lib/src/widgets/image_message_view.dart

```diff
@@ -36,12 +36,16 @@ class ImageMessageView extends StatelessWidget {
     required this.isMessageBySender,
     this.imageMessageConfig,
     this.messageReactionConfig,
+    this.highlightImage = false,
+    this.highlightScale = 1.2,
```

Comment:
Suggestion: please try highlightScale with value 1.1 as i observed in video it looks pretty odd.